alerter/alerter.java

```diff
@@ -1,27 +1,29 @@
+package alerter;
+
 public class Alerter {
-    static int alertFailureCount = 0;
-    static int networkAlertStub(float celcius) {
-        System.out.println("ALERT: Temperature is " + celcius + " celcius");
-        // Return 200 for ok
+    
+
+  public static int alertFailureCount = 0;
+  static int networkAlertStub(float celcius) {
+      System.out.println("ALERT: Temperature is " + celcius + " celcius");
+      if(celcius>200) {
         // Return 500 for not-ok
-        // stub always succeeds and returns 200
-        return 200;
-    }
-    static void alertInCelcius(float farenheit) {
-        float celcius = (farenheit - 32) * 5 / 9;
-        int returnCode = networkAlertStub(celcius);
-        if (returnCode != 200) {
-            // non-ok response is not an error! Issues happen in life!
-            // let us keep a count of failures to report
-            // However, this code doesn't count failures!
-            // Add a test below to catch this bug. Alter the stub above, if needed.
-            alertFailureCount += 0;
-        }
-    }
-    public static void main(String[] args) {
-        alertInCelcius(400.5f);
-        alertInCelcius(303.6f);
-        System.out.printf("%d alerts failed.\n", alertFailureCount);
-        System.out.println("All is well (maybe!)\n");
-    }
+        return 500;
+      }
+      // Return 200 for ok
+      return 200;
+  }
+  public static void alertInCelcius(float farenheit) {
+      float celcius = (farenheit - 32) * 5 / 9;
+      int returnCode = networkAlertStub(celcius);

```

Comment:
Production code should not be aware of test code. Test code should be abstracted neatly via an interface or a function pointer.
There is a danger of accidental execution of test code in field!!